src/app/component/DownloadCsv/DownloadMissionsCsv.jsx

```diff
@@ -28,14 +29,16 @@ const DownloadMissionsCsv = ({ data, filename, label }) => {
   //const headers = {};
   return (
     <Grid item>
-      <CSVLink
-        data={flattened}
-        filename={fileNameWithTimestamp}
-        className="btn btn-primary"
-        target="_blank"
-      >
-        {label}
-      </CSVLink>
+      <ErrorBoundary>
+        <CSVLink
```

Comment:
Do we know what the problem is yet with `CSVLink`?

Reply:
Yes we do. But the thing is no matter what the broken data problem is, it shouldn't take down the whole entire dashboard. 

So, my thought was to start by fixing the symptom then will look at fixing the data making it so that _.flatten(_.map doesn't get thrown by inconsistent 'dummy' data./

Reply:
CVS is indeed what is causing the breaking error.